src/i18n/Localizable.xcstrings

```diff
@@ -16263,6 +16348,7 @@
       }
     },
     "Pad" : {
+      "comment" : "a bit ambiguous word. example needed :(",
```

Comment:
"Pad" is a kind of joystick device. Here "Pad" is related to the four directional keys. "Pad" isn't the best word I think.